pkgs/by-name/ap/apidog/package.nix

```diff
@@ -0,0 +1,62 @@
+{
+  pkgs,
+  fetchzip,
+  appimageTools,
+  makeWrapper,
+  imagemagick,
+  lib,
+  ...
+}:
+let
+  pname = "apidog";
+  version = "2.6.28";
+  name = "${pname}-${version}";
+
+  # First fetch the zip file
+  zipFile = fetchzip {
+    url = "https://file-assets.apidog.com/download/Apidog-linux-latest.zip";
+    sha256 = "sha256-8Lhyt3GkwBKBpdyd1phqDRPVsBZUwUNVpzG8UlaevwI=";
```

Comment:
this hash no longer seems to match the latest zip you specified. would they maybe offer zips of fixed versions as well?

Reply:
Looks like they are constantly updating the hash even though the package is the same. From what I've seen, they don't offer fixed versions for the zip files.

Reply:
i've seen another package use archive.org links to pin contents at a certain point in time. i'd wonder if that could maybe pose a solution here as well, so the package wouldn't just break like a week down the line.

Reply:
Oh yeah, that's a good idea. [Badlion-client](https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/games/badlion-client/default.nix#L27) is doing exactly that, I'll take a better look into archive.org.

Reply:
@KiaraGrouwstra I just pushed the changes for the archive.org url :+1: